recipes/autocfg/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "autocfg" %}
+{% set version = "0.0.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/autocfg-{{ version }}.tar.gz
+  sha256: 749986b4f3b3bd85b15298734bf8fa4a590e6c34a314ac515025e058ed76c319
+
+build:
+  skip: true  # [py2k]
```

Comment:
```suggestion
```
CF don't build for legacy python to this isn't required.
Upstream specify `py36` as the minimum so you should specify that constraint instead:
https://github.com/zhreshold/autocfg/blob/7fb8311bfaea350e8bdcc7010e2a19c9859989d4/setup.py#L21

Reply:
@dhirschfeld This line has been removed as suggested. Thanks.